.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
[Optional] I suggest that you ignore this `.vscode` folder since it's only a local setup for your code editor. This is so that your project is less cluttered.